app/src/main/res/values/dimens.xml

```diff
@@ -1,26 +1,26 @@
 <resources>
     <!-- Default screen margins, per the Android Design guidelines. -->
     <!-- margin specifics -->
-    <dimen name="layout_margin_extra_small">2dp</dimen>
-    <dimen name="layout_margin_small">6dp</dimen>
+<!--    <dimen name="layout_margin_extra_small">2dp</dimen>-->
+<!--    <dimen name="layout_margin_small">6dp</dimen>-->
     <dimen name="layout_margin_medium">8dp</dimen>
     <dimen name="layout_margin_moderate">10dp</dimen>
     <dimen name="layout_margin_large">16dp</dimen>
     <dimen name="layout_margin_extra_large">32dp</dimen>
-    <dimen name="layout_margin_extreme_large">56dp</dimen>
+<!--    <dimen name="layout_margin_extreme_large">56dp</dimen>-->
     <dimen name="layout_margin_huge">120dp</dimen>
 
     <!-- padding specifics -->
-    <dimen name="padding_extra_small">2dp</dimen>
+<!--    <dimen name="padding_extra_small">2dp</dimen>-->
     <dimen name="padding_small">6dp</dimen>
     <dimen name="padding_medium">8dp</dimen>
     <dimen name="padding_large">16dp</dimen>
-    <dimen name="padding_intermediate">24dp</dimen>
+<!--    <dimen name="padding_intermediate">24dp</dimen>-->
     <dimen name="padding_extra_large">32dp</dimen>
     <dimen name="padding_extra_larger">64dp</dimen>
-    <dimen name="padding_mem_graph">54dp</dimen>
+<!--    <dimen name="padding_mem_graph">54dp</dimen>-->
     <dimen name="margin_large">60dp</dimen>
-    <dimen name="margin_extra_large">248dp</dimen>
+<!--    <dimen name="margin_extra_large">248dp</dimen>-->
```

Comment:
Removing unused resources doesn't mean commenting them out. Please check well if a `res` is used anywhere in a redundant way i.e, the same resource value is kept with different names, remove them. Also be sure to change in the codebase/XML files where those removed resources were used.

Reply:
Ok yeah forgot to delete the comments. I'll update soon.

Reply:
There are some unused declarations. I'm confused if i should remove them or they are for future use.

Reply:
If they are unused, just remove them. In future, we will create if needed.